widget/playground/src/constants/presets.ts

```diff
@@ -279,20 +279,6 @@ export const WIDGET_COLORS: { key: WidgetColorsKeys; label: string }[] = [
   },
 ];
 export const DEFAULT_COLORS = {
-  dark: {
-    primary: undefined,
-    secondary: undefined,
-    neutral: undefined,
-    background: undefined,
-    foreground: undefined,
-    info: undefined,
-  },
-  light: {
-    primary: undefined,
-    secondary: undefined,
-    neutral: undefined,
-    background: undefined,
-    foreground: undefined,
-    info: undefined,
-  },
+  dark: PRESETS.find((preset) => preset.id === 1)?.dark as WidgetColors,
```

Comment:
it can be `undefined` (based on `find(...)?`) but you have added a type assertion which eliminated the `undefined`.

Reply:
Please also don't use magic values which means we prefer `const DEFAULT_PRESET_ID = 1;` over using a magic number directly.

Reply:
Thanks, I changed my code to use the "DEFAULT_PRESET_ID" that you suggested. About the other comment, the preset with id equal to 1 is hard coded so it is not possible for the result of the find method to be 'undefined' so I don't get the problem here.